content/pages/en.yaml

```diff
@@ -510,6 +510,518 @@ sections:
+                solution_code: |
+                    icecream = ['starwberry', 'chocolate']
+                    print 'I love ' icecream[random] ' icecream'
+        -   title: "Students use the wrong brackets"
+            example:
+                error_text: |
+                    From this level on lists should be in brackets.
+                error_code: |
+                    icecream = ('starwberry', 'chocolate'}
+                    print 'I love ' icecream[random] ' icecream'
+                solution_text: |
+                    This is the correct code:
+                solution_code: |
+                    icecream = ['starwberry', 'chocolate']
+                    print 'I love ' icecream[random] ' icecream'
+        -   title: "Students forget the quotation marks while focussing on the brackets"
+            example:
+                error_text: |
+                    Students are sometimes very focussed on the new aspect of the syntax, that they forget the quotation marks.
+                error_code: |
+                    icecream = [starwberry, chocolate]
+                    print 'I love ' icecream[random] ' icecream'
+                solution_text: |
+                    This is the correct code:
+                solution_code: |
+                    icecream = ['starwberry', 'chocolate']
+                    print 'I love ' icecream[random] ' icecream'
+        -   title: "Students still use the old at random command"
+            example:
+                error_text: |
+                    Students are sometimes very focussed on the new aspect of the syntax, that they forget the quotation marks.
+                error_code: |
+                    icecream = [starwberry, chocolate]
+                    print 'I love ' icecream at random ' icecream'
+                solution_text: |
+                    This is the correct code:
+                solution_code: |
+                    icecream = ['starwberry', 'chocolate']
+                    print 'I love ' icecream[random] ' icecream'
+        -   title: "Students forget the quotation marks while focussing on the brackets"
+            example:
+                error_text: |
+                    Students are sometimes very focussed on the new aspect of the syntax, that they forget the quotation marks.
+                error_code: |
+                    icecream = [starwberry, chocolate]
+                    print 'I love ' icecream[random] ' icecream'
+                solution_text: |
+                    This is the correct code:
+                solution_code: |
+                    icecream = ['starwberry', 'chocolate']
+                    print 'I love ' icecream[random] ' icecream'
+    -   level: '17'
+        sections:
+        -   title: "Students use elif like else, so without a condition"
+            example:
+                error_text: |
+                    Make sure that the students know to use both the full repeat command and the print command.
```

Comment:
This error text is not related to the title nor the error code.

Reply:
fixed